chat-client/src/client/mynahUi.ts

```diff
@@ -299,8 +299,11 @@ export const createMynahUi = (
     }
 
     const addChatResponse = (chatResult: ChatResult, tabId: string, isPartialResult: boolean) => {
+        const { type, ...chatResultWithoutType } = chatResult
+
         if (isPartialResult) {
-            mynahUi.updateLastChatAnswer(tabId, { ...chatResult })
+            // type for MynahUI differs from ChatResult types so we ignore it
+            mynahUi.updateLastChatAnswer(tabId, { ...chatResultWithoutType })
```

Comment:
Alternatively, we could map what we get from protocol to mynahUI `type`. Then, we would enable prompts for based chat streaming too. I do not think it's something that will be used in practice, but it's a legit feature to have in general.